src/qt/pivx/mnmodel.cpp

```diff
@@ -31,7 +31,7 @@ void MNModel::updateMNList()
         if (!pmn) {
             pmn = new CMasternode();
             pmn->vin = CTxIn();
-            pmn->activeState = CMasternode::MASTERNODE_MISSING;
+            pmn->activeState = CMasternode::MASTERNODE_REMOVE;
```

Comment:
This isn't a "remove", this is a "not found". It happens when the entry is in the config file only, without broadcasting the start message.
The `ColumnIndex::STATUS` data is already contemplating this actually, we can leave it as nullptr and remove this whole if block. No need to create an empty masternode here.

Reply:
Yea, `MASTERNODE_MISSING` doesn't even exist anymore. I've put it `MASTERNODE_REMOVE` just as placeholder, it's going to go away in any case with the removal of `activeState`.  

But there is, actually, another bug, introduced here... setting `pmn->vin = CTxIn()`, which will leave the entry without collateral txid-txidn. Will revert that.